megalinter/descriptors/json.megalinter-descriptor.yml

```diff
@@ -217,3 +217,22 @@ linters:
       vscode:
         - name: prettier-vscode
           url: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
+  # npm-package-json-lint
+  - linter_name: npm-package-json-lint
+    linter_url: https://npmpackagejsonlint.org/
+    linter_repo: https://github.com/tclindner/npm-package-json-lint
+    linter_rules_url: https://npmpackagejsonlint.org/docs/rules
+    linter_rules_configuration_url: https://npmpackagejsonlint.org/docs/configuration
+    cli_lint_mode: list_of_files
```

Comment:
Add :
```yaml
file_names_regex:
  - "package\\.json"
```

Reply:
But this property is not valid inside a linter, is it? According to the schema it can only be placed in the top level of the descriptor.

Reply:
Mmm if it's not in the schema it should be added, to allow override of such property from descriptor to linter level
file_extensions should also be overridden to [] because we don't want to take all json files ^^

Reply:
PR created: https://github.com/oxsecurity/megalinter/pull/2163